b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/roleItemDetailsSection/general.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from "react";
+import { Field, Form } from "react-final-form";
+import { Button, ButtonToolbar, Loader, useToaster } from "rsuite";
+import FormSuite from "rsuite/Form";
+import styles from "../../../../../../../styles/Settings.module.css";
+import decodePatchRole from "../../../../../../../util/apiDecode/settings/role/decodePatchRole";
+import { PatchMethod, checkIfJSONisEmpty } from "../../../../../../../util/util/common/common";
+import { LOADING_DISPLAY_BLOCK, LOADING_DISPLAY_NONE } from "../../../../../../../util/util/frontendUtil/frontendUtil";
+import { errorTypeDialog, successTypeDialog } from "../../../../../../common/dialog";
+import HelperText from "../../../../../../common/helperText";
+
+/**
+ * 
+ * @param prop - `fetchData` - function , `session`, `roleDetails` - Object
+ * 
+ * @returns The general section of role details
+ */
+export default function General(prop) {
+
+    const { fetchData, session, roleDetails } = prop;
+
+    const [ loadingDisplay, setLoadingDisplay ] = useState(LOADING_DISPLAY_NONE);
+
+    const toaster = useToaster();
+
+    const nameValidate = (name, errors) => {
+        if (!name) {
+            errors.name = "This field cannot be empty";
+        }
+
+        return errors;
+    };
+
+
+    const validate = values => {
+        let errors = {};
+
+        errors = nameValidate(values.name, errors);
+
+        return errors;
+    };
+
+    const onDataSubmit = (response, form) => {
+        if (response) {
```

Comment:
make the code consistent, by leaving a new line after the declaration. Check other places too.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/83fabfed8c8201a0b1da8e023dbca5eb6494c56b